### PR TITLE
Adding example of why enum inheritance is not allowed.

### PR DESCRIPTION
**Pull request by Danack**
Hoping the CI works to test it's formatted okay....

### CODE REVIEW

File: language/enumerations.xml [hunk 1]
```diff
@@ -727,6 +727,103 @@ Baz Enum:int {
...
+    foo($b);
+}
+?>
+]]>
+ </programlisting>
+
+ <simpara>
+  This code is type-safe, as B follows the contract of A, and through the magic of
+  co/contra-variance, any expectation you may have of the methods will be
+  preserved, exceptions excepted.
```
Comment (Girgias) [inline language/enumerations.xml:757]: The manual shouldn't use personalisation (i.e. using "you")
So maybe something like:
```suggestion
  This code is type-safe, as B follows the contract of A, and through the magic of
  co/contra-variance, any expectation that is hold about methods will be
  preserved, exceptions excepted.
```
?

**Review by Girgias — COMMENTED** [review]
The indentation is also somewhat off at the beginning where it should have 2 spaces instead of 1

File: language/enumerations.xml (hunk 1 above)
Comment (Danack) [reply language/enumerations.xml]: "The manual shouldn't use personalisation"

Kind of hard to refer to someone's state of mind without doing that.....impersonal pronoun it is then.

**Review by Girgias — APPROVED** [review]
Thank you!

File: language/enumerations.xml [hunk 2]
```diff
@@ -727,6 +727,103 @@ Baz Enum:int {
...
+  <simpara>
+   Classes have contracts on their methods:
+  </simpara>
+
+  <programlisting role="php">
+<![CDATA[
+<?php
+
+class A {}
+class B {}
```
Comment (TimWolla) [inline language/enumerations.xml:743]: Is extends missing here?

**Pull request merged**